packages/api/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "baseUrl": "./src",
     "declaration": true,
     "esModuleInterop": true,
-    "lib": ["dom", "es2020"],
+    "lib": ["dom", "dom.iterable", "es2020"],
```

Comment:
?

Reply:
I believe I added this because I'm doing some iteration on the `Headers` API in tests and TS doesn't think that `Headers.entries()` exists without this library being present.